spec/fixtures/modules/add_group/plans/init.pp

```diff
@@ -0,0 +1,26 @@
+plan add_group (TargetSpec $nodes) {
+  $new_target = Target.new(
+    "0.0.0.0:20024",
+    "host-key-check" => false,
+    "user" => 'bolt',
+    'password' => 'bolt'
+  )
+  # Add new facts/var, specifically one that is novel, the other that should override existing.
+  $new_target.add_facts({'plan_context' => 'keep', 'override_parent' => 'keep'})
+  $new_target.set_var('plan_context','keep')
+  $new_target.set_var('override_parent', 'keep')
+  # Add from Target object
+  $new_target.add_to_group('add_me')
+  # Add from string
+  add_to_group('bar_1', 'bar')
```

Comment:
Is there any verification that this worked?

Reply:
Yep last expect under `'computes facts and vars based on group hierarchy'`.